Lib/javascript/v8/javascriptrun.swg

```diff
@@ -370,9 +373,15 @@ fail:
 
 class SwigV8PackedData {
 public:
-  SwigV8PackedData(void *data, size_t size, swig_type_info *type): data(data), size(size), type(type) {};
+  SwigV8PackedData(void *data, size_t size, swig_type_info *type): data(nullptr), size(size), type(type) {
+    this->data = malloc(size);
+    if (this->data != nullptr)
+      memcpy(this->data, data, size);
+  };
 
   ~SwigV8PackedData() {
+    if (this->data)
+      free(this->data);
```

Comment:
There's no need to NULL-check before calling `free()` - ANSI/ISO C requires `free()` accept `NULL`.